src/components/general/ChangePasswordModal/index.tsx

```diff
@@ -0,0 +1,162 @@
+import React, { useCallback } from 'react';
+import { isDefined } from '@togglecorp/fujs';
+import {
+    useForm,
+    ObjectSchema,
+    PartialForm,
+    requiredCondition,
+    lengthGreaterThanCondition,
+    lengthSmallerThanCondition,
+} from '@togglecorp/toggle-form';
+import {
+    Button,
+    Modal,
+    TextInput,
+} from '@the-deep/deep-ui';
+import NonFieldError from '#components/ui/NonFieldError';
+import { useLazyRequest } from '#utils/request';
+import _ts from '#ts';
+import { User } from '#typings';
+
+import styles from './styles.scss';
+
+type FormType = {
+    oldPassword?: string;
+    newPassword?: string;
+    confirmPassword?: string;
+}
+
+type FormSchema = ObjectSchema<PartialForm<FormType>>;
+type FormSchemaFields = ReturnType<FormSchema['fields']>;
+
+const changePasswordSchema: FormSchema = {
+    fields: (): FormSchemaFields => ({
+        oldPassword: [requiredCondition],
+        newPassword: [
+            lengthGreaterThanCondition(4),
+            lengthSmallerThanCondition(129),
```

Comment:
I think this is 120 right? Could you confirm what is done in the backend.
@k9845 ?

Reply:
```suggestion
            lengthGreaterThanCondition(8),
            lengthSmallerThanCondition(128),
```
This should be according to backend